### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by memiscavage**

### CODE REVIEW

File: views/homepage.handlebars [hunk 1]
```diff
@@ -3,9 +3,15 @@
...
+    <link rel="stylesheet" href="/css/homepage.css">
 </head>
 <body>
+    <div class="container">
+        <h1>Welcome to mudi!</h1>
+        <h3>Track your daily mood here</h3>
+    <img src="/images/mudi_logo.png" alt="Logo for mudi">
+    </div>
     {{{body}}}
     stuff
```
Comment (clendinning1) [inline views/homepage.handlebars:16]: lol

Comment (memiscavage) [reply views/homepage.handlebars:16]: lmao

**Review by clendinning1 — CHANGES_REQUESTED** [review]
get rid of stuff lol

**Comment by memiscavage** [comment]
ok try again por favor

**Review by clendinning1 — APPROVED** [review]
yes

**Pull request merged**